### ISSUE TITLE
iterations when converging

### ISSUE THREAD

**Issue by cxy0616** [MLEnthusiast/MHCLN#5]
Did you iterate the network model 10,000 times on the UCMD and AID datasets? If not, can you tell me the number of iterations when converging on these two data sets?

**Comment by MLEnthusiast** [comment]
If I remember correctly I used 25000 iterations for AID since the dataset is larger than UCMD

**Comment by cxy0616** [comment]
Thank you for your reply. I think the selection method of triples in the code is randomly selected offline. Each batch_size has 30 triples. Have you tried to select batch_all or batch_hard online?

**Comment by MLEnthusiast** [comment]
Yes they are always randomly selected. But it's an interesting suggestion to do hard mining. Please let me know if you find improvements using batch_all or batch_hard. Thanks.
